Sources/MediaPublicTracker.swift

```diff
@@ -182,14 +182,14 @@ class MediaPublicTracker: MediaTracker {
         timer = nil
     }
 
-    func getCurrentTimeStamp() -> Int64 {
+    func getCurrentTimeStamp() -> Int {
         return Date().millisecondsSince1970
     }
 }
 
 private extension Date {
-    var millisecondsSince1970: Int64 {
-        return Int64((timeIntervalSince1970 * 1000.0).rounded())
+    var millisecondsSince1970: Int {
+        return Int((timeIntervalSince1970 * 1000.0).rounded())
```

Comment:
This change will cause problems in edge cases where a user is running on a 32-bit system (as Apple is stamping out the support of 32-bit systems, this is an edge case). The [`Int`](https://developer.apple.com/documentation/swift/int) type is just a wrapper around `Int32` and `Int64` where on 32-bit systems `Int` is 4 bytes while on 64-bit systems `Int` is 8 bytes . The problem here is `timeIntervalSince1970 * 1000.0` is larger than `Int32`, so on a 32-bit system this code will crash or simply give the wrong result. 
When handling Dates and times, we should continue to use `TimeInterval` as much as possible, and use `Int64` if we need to convert to an `Int`.  Timestamps which are not Dates, such as "playhead" or a tick interval or timeout, can still use `Int` as those values are not expected to be very large, especially since "playhead" is at most 24 hours in seconds.